guides/source/form_helpers.md

```diff
@@ -715,48 +796,64 @@ Output:
 Uploading Files
 ---------------
 
-A common task is uploading some sort of file, whether it's a picture of a person or a CSV file containing data to process. File upload fields can be rendered with the [`file_field`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-file_field) helper.
+A common task with forms is allowing users to upload a file. It could be an avatar image or a CSV file with data to process. File upload fields can be rendered with the [`file_field`](https://api.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-file_field) helper.
```

Comment:
Might be worth briefly mentioning the `:multiple` option for `file_field`.

Reply:
`:multiple` and other options are described in the linked API docs as well.
Is it that important?

Reply:
Probably not super important, tbh.